src/components/moves/typeFields/SimpleFields.js

```diff
@@ -41,6 +41,14 @@ export default class SimpleFields extends React.Component {
     isTypeInfoOpen: true
   };
 
+  userGuildOptions = () => {
+    let guildOptions = [{ label: 'Public', value: '1' }];
```

Comment:
@brian-frederick I think I know the answer to this, but these `guildOptions` will eventually pull from listed Guilds in the db tied to user accounts? Am I tracking that right?

Reply:
yep! well it already does actually. check out the screen shot below from my branch. My concept for this was that in addition to associating a move with your guild (server), you can also just make it public. I am beginning to re-think question/this though so maybe you, me, and chris can talk through the business logic at some point? 

The move guild inits to public as you're pointing out in MoveCreate.js and allows for changing to something else by enriching the base option with the user's guilds. The guild names you see below are pulled directly from discord user api and as you can see see top right i'm signed in!

![image](https://user-images.githubusercontent.com/21226878/99920248-d3f44280-2ce7-11eb-9a01-505de8e90bb9.png)

Reply:
That all makes a lot of sense and is extremely exciting!